client/topbar.ts

```diff
@@ -0,0 +1,37 @@
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
```

Comment:
These two lines are included in every example of the angular2-meteor tutorial, I first thought it's just part of the project. However I tested and nothing changed when I took them out. I guess we can leave them out for now.